pycaching/geocaching.py

```diff
@@ -48,10 +50,31 @@ def _request(self, url, *, expect="soup", method="GET", login_check=True, **kwar
         except requests.exceptions.RequestException as e:
             raise Error("Cannot load page: {}".format(url)) from e
 
-    def login(self, username, password):
+    def login(self, username=None, password=None):
         """Logs the user in.
 
         Downloads the relevant cookies to keep the user logged in."""
+        if username is None or password is None:
```

Comment:
An what about your tests, you are testing that login("","") must raise an exception. Is change to login("0", "0") and hope that user "0" will not have password "0" OK?